_overviews/scala-book/prelude-taste-of-scala.md

```diff
@@ -462,33 +462,66 @@ There are many (many!) more methods available to Scala collections classes, and
 
 ## Tuples
 
-Tuples let you put a heterogenous collection of elements in a little container. Tuples can contain between two and 22 values, and they can all be different types. For example, given a `Person` class like this:
+Tuples let you put a heterogenous collection of elements in a little container. A tuple can contain between two and 22 values, and all of the values can have different types. For example, this is a tuple that holds three different types, an `Int`, a `Double`, and a `String`:
 
 ```scala
-class Person(var name: String)
+(11, 11.0, "Eleven")
 ```
 
-You can create a tuple that contains three different types like this:
+This is known as a `Tuple3`, because it contains three elements.
+
+Tuples are convenient in many places, such as where you might use an ad-hoc class in other languages. For instance, you can return a tuple from a method instead of returning a class:
+
+```scala
+def getAaplInfo() = {
```

Comment:
I suggest showing the result type for two reasons. First, this is good practice, second, it shows the type syntax for tuples.

```suggestion
def getAaplInfo(): (String, BigDecimal, Long) = {
```

Reply:
I debated about doing that, so yes, that’s great. Thanks!